marlowe-playground-client/src/Web/HTML/WindowExtra.js

```diff
@@ -0,0 +1,7 @@
+exports._close = function (window) {
+  window.close();
+};
+
+exports._postMessage = function (message, window) {
+  window.postMessage(message);
+};
```

Comment:
could you move WindowExtra to `./playground-common`

Reply:
done